sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -224,6 +224,30 @@ class annotated_arg {
     // instances of propertyT
     template<typename propertyT>
     static constexpr /*unspecified*/ get_property();
+
+  template <typename T2, typename PropertyList2>
+  auto operator+(const annotated_arg<T2, PropertyList2> &other) const;
```

Comment:
This isn't useful when I want to add an annotated_arg<T> to a T.  See my comment in Greg's chain for a solution to that.